agdk/game_mode/app/src/main/AndroidManifest.xml

```diff
@@ -41,13 +45,16 @@
         <category android:name="android.intent.category.LAUNCHER" />
       </intent-filter>
     </activity>
-    <!-- Enabling Game Modes -->
-    <meta-data
+
+    <!-- Enable this meta-data chunk to enable Game Mode API if you're targeting API Level 31 -->
+    <!-- Remove it if you are targeting API Level 33 (recommended) -->
```

Comment:
What happens if the targetSDK version is let's say 33 and the device's API version is 30, does the manifest for the API 33 still work on the device?

Reply:
No it will not work on runtime, but it will still be compilable as long as compileSdkVersion is 31 or higher; when getGameMode function is called on runtime it will return "UNSUPPORTED"

Reply:
Got it. Then how about some thing like this,
1) Move line 44-57 right after to line 33.
Update comment to something like this:

<!-- Enabling Game Modes -->
<!-- Here is how to enable Game Mode API if you're targeting API Level 33 (recommended) -->
<meta-data android:name="android.game_mode_config" android:resource="@xml/game_mode_config" />

<!-- Use meta data below instead if you are targeting API level 31/32—>
<!-- <meta-data
   android:name="com.android.app.gamemode.performance.enabled"
   android:value="true"/>
 <meta-data android:name="com.android.app.gamemode.battery.enabled"
   android:value="true"/> -->

Reply:
Updated, thanks @hak !